### PR TITLE
chore(releasing): Prepare v0.32.1

### PR DESCRIPTION
**Pull request by jszwedko**
Checklist: https://github.com/vectordotdev/vector/issues/18312

### CODE REVIEW

File: website/cue/reference/releases/0.32.1.cue [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+	whats_next: []
+	description: """
+		This patch release contains a fix for a regression in 0.32.0 and fixes a few issues with the release artifacts.
+
+		**Note:** Please see the release notes for [`v0.32.0`](/releases/0.32.0/) for additional changes if upgrading from
+		`v0.31.X`. In particular, the upgrade guide for breaking changes.
+		"""
+
+	changelog: [
```
Comment (dsmith3197) [inline website/cue/reference/releases/0.32.1.cue:15]: Is it worth also calling out the otel max payload size adjustment here as well?

**Review by dsmith3197 — APPROVED** [review]
lgtm

File: website/cue/reference/releases/0.32.1.cue (hunk 1 above)
Comment (jszwedko) [reply website/cue/reference/releases/0.32.1.cue:15]: Thanks for catching that. I could have sworn I added a note for it, but it seems like I didn't.

**Review by dsmith3197 — APPROVED** [review]

**Pull request merged**